OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -73,6 +73,9 @@
     <StartAction>Project</StartAction>
     <StartArguments>Game.Mod=ts</StartArguments>
   </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
```

Comment:
The app.manifest is embedded in the assembly so we don't need to update packaging scripts. When I run make.cmd all, it is embedded so I assume we don't need to change makefile, it is sufficient that it is included in .csproj file.